ts/Series/Column/ColumnSeries.ts

```diff
@@ -822,7 +823,8 @@ class ColumnSeries extends Series {
                                     indexInCategory = totalInCategory;
                                 }
                                 if (stackItem.hasValidPoints) {
-                                    totalInCategory++;
+                                    reversedStacks ?
```

Comment:
What do you think about adding a comment with a ticket number here so the ticket can be easier to find in case of any problems in the future?